src/System.Private.CoreLib/shared/System/Number.Parsing.cs

```diff
@@ -787,20 +782,14 @@ internal static bool TryParseInt32IntegerStyle(ReadOnlySpan<char> value, NumberS
 
                 // Potential overflow now processing the 19th digit.
                 if ((uint)index >= (uint)value.Length)
-                    goto DoneAtEndButPotentialOverflow;
+                    goto DoneAtEnd;
```

Comment:
Nit: I realize this is left-over from my changes, but maybe this if block should be moved up above the comment?  Otherwise it's confusing why we're saying there's now a potential overflow but then goto the DoneAtEnd label that doesn't account for overflow.  Same in other similar occurrences.